iphone/Classes/MediaModule.m

```diff
@@ -1441,6 +1441,9 @@ -(void)sendPickerSuccess:(id)event
 		[NSThread detachNewThreadSelector:@selector(dispatchCallback:) toTarget:self withObject:[NSArray arrayWithObjects:@"success",event,listener,nil]];
 #else
 		[self dispatchCallback:@[@"success",event,listener]];
+        	//TIMOB-24389 : As memory flushing was taking more time, so memory was growing continuously causing crash of app.
+        	KrollContext *krollContext = [self.pageContext krollContext];
+        	[krollContext forceGarbageCollectNow];
```

Comment:
This is still only executed in `sendPickerSuccess`. The method `saveCompletedForImage:` and others create a blob as well, are there no leaks there? And change the comment to this:
```objc
// TIMOB-24389: Force the heap to be GC'd to avoid Ti.Blob references to be dumped.
```